### PR TITLE
Fix For 'GameDig' - Store Game Info Mod

### PR DESCRIPTION
**Pull request by Harley2787**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by DetroitWhiskey136** [comment]
You have a couple linting issues to resolve before i can approve this.

File: actions/game_server_MOD.js
Comment (DetroitWhiskey136) [inline actions/game_server_MOD.js:417]:
```suggestion
    const { GameDig } = require('gamedig');
```

File: actions/game_server_MOD.js
Comment (DetroitWhiskey136) [inline actions/game_server_MOD.js:416]:
```suggestion
```

File: actions/game_server_MOD.js
Comment (DetroitWhiskey136) [inline actions/game_server_MOD.js:421]:
```suggestion

```

**Review by DetroitWhiskey136 — CHANGES_REQUESTED** [review]

File: actions/game_server_MOD.js
Comment (DetroitWhiskey136) [reply actions/game_server_MOD.js:416]: this can be removed since its not being used anymore., I'll leave that up to you though.

Comment (Harley2787) [reply actions/game_server_MOD.js]: Removed that now

**Review by DetroitWhiskey136 — APPROVED** [review]

**Pull request merged**
